magento/magento-cloud-docker
# MAGECLOUD-5351: WEB docker container does not work in production mode

### Description

NGINX does not start if `fpm_xdebug` is not available 
```
Attaching to magento-cloud_web_1
web_1            | 2020/02/18 16:55:21 [emerg] 13#13: host not found in upstream "fpm_xdebug:9000" in /etc/nginx/conf.d/default.conf:10
web_1            | nginx: [emerg] host not found in upstream "fpm_xdebug:9000" in /etc/nginx/conf.d/default.conf:10
web_1            | nginx: configuration file /etc/nginx/nginx.conf test failed
web_1            | 2020/02/18 16:55:21 [emerg] 1#1: host not found in upstream "fpm_xdebug:9000" in /etc/nginx/conf.d/default.conf:10
web_1            | nginx: [emerg] host not found in upstream "fpm_xdebug:9000" in /etc/nginx/conf.d/default.conf:10
```

### Fixed Issues (if relevant)

1. https://magento2.atlassian.net/browse/MAGECLOUD-5351

### Manual testing scenarios

1. Check developer mode with xdebug
2. Check developer mode without xdebug
3. Check production mode with xdebug
4. Check production mode without xdebug

### Contribution checklist
